_episodes/11-connecting.md

```diff
@@ -0,0 +1,427 @@
+---
+title: "Connecting to a remote HPC system"
+teaching: 25
+exercises: 10
+questions:
+- "How do I log in to a remote HPC system?"
+objectives:
+- "Configure secure access to a remote HPC system."
+- "Connect to a remote HPC system."
+keypoints:
+- "An HPC system is a set of networked machines."
+- "HPC systems typically provide login nodes and a set of worker nodes."
+- "The resources found on independent (worker) nodes can vary in volume and
+  type (amount of RAM, processor architecture, availability of network mounted
+  filesystems, etc.)."
+- "Files saved on one node are available on all nodes."
+---
+
+## Secure Connections
+
+The first step in using a cluster is to establish a connection from our laptop
+to the cluster. When we are sitting at a computer (or standing, or holding it
+in our hands or on our wrists), we have come to expect a visual display with
+icons, widgets, and perhaps some windows or applications: a graphical user
+interface, or GUI. Since computer clusters are remote resources that we connect
+to over often slow or laggy interfaces (WiFi and VPNs especially), it is more
+practical to use a command-line interface, or CLI, in which commands and
+results are transmitted via text, only. Anything other than text (images, for
+example) must be written to disk and opened with a separate program.
```

Comment:
Behold, a rabbit hole! What's written here (and in the suggested edit) is not wrong, but the phrasing is imprecise and absolutist.
* While a vanilla SSH connection can only display text, X forwarding is built-in to most clients. Image viewers, music players, and application GUIs _can_ be opened from the terminal, and some software (_ranger_) can display images on the terminal anyway.
* Images, data files, video, etc. has to be written to memory (disk or RAM) _somewhere_, and transferred _somehow_, possibly coming through an intermediate application, before arriving intelligibly on the local monitor.

Since this whole process can be accomplished in many creative ways, I suggest revising this to soften the language and more clearly state what we're trying to say -- which is that, for the purposes of this lesson, we're only going to work with text-based data in a vanilla shell session.

Reply:
Attempted to address in 1967dee.